### DISCUSSION TITLE
About Openthread Protocol Stack Multiple Handles

### DISCUSSION THREAD

**Discussion by tuopu** [openthread/openthread#9430] [Q&A] [answered]
Does the multi handle of openthread refer to the ability to control multiple RF devices? If so, enable OPENTHREAD_ CONFIG_ MULTIPLE_ INSTANCE_ How is the handle associated with the RF device in the ENABLE case.

**Comment by jwhui** [comment, marked as answer]
OpenThread supports multiple instances, each instance supported by a single IEEE 802.15.4 radio.

The OpenThead APIs include an `otInstance` parameter to indicate the instance. Check out the [otInstance*](https://openthread.io/reference/group/api-instance) APIs for more information.
